tox.ini

```diff
@@ -33,6 +33,8 @@ setenv =
     py39: INTERP=py39
     pypy3: INTERP=pypy3
     nocov: PYTHONASYNCIODEBUG=1
+    !py39: PY_39=lt
+    py39: PY_39=ge
```

Comment:
These two lines work together to mark whether the Python interpreter is < 3.9 (`lt`) or >= 3.9 (`ge`); this env var will be used later in `.coveragerc`